app/assets/stylesheets/views/_homepage.scss

```diff
@@ -102,16 +102,17 @@
   padding: 0;
 
   @include govuk-media-query($from: desktop) {
-    @include columns($items: 10, $columns: 2, $selector: "li", $flow: column);
+    max-width: 66.66%;
+    columns: 2;
+    column-gap: 15px;
```

Comment:
I checked on [caniuse?](https://caniuse.com/?search=column-gap) and it seems `column-gap` isn't supported by IE11. I was thinking an alternative approach could be to separate the unordered list of 10 items into two unordered lists of 5 items and then use the same govuk-grid-column classes that are being used in the row above?